src/prefect/testing/cli.py

```diff
@@ -75,6 +78,7 @@ def invoke_and_assert(
         temp_dir: if provided, the CLI command will be run with this as its present
             working directory.
     """
+    prompts_and_responses = prompts_and_responses or []
```

Comment:
avoid mutable defaults

Reply:
😬 i'm surprised that lasted this long! good catch